src/backend/metal/RenderPipelineMTL.mm

```diff
@@ -248,7 +247,7 @@ MTLVertexFormat toMTLVertexFormat(VertexFormat vertexFormat)
         info += StringUtils::BlendFactor2String(blendDescriptor.destinationAlphaBlendFactor);
     }
     
-    return HashAlgorithm::PJWHash(info.c_str(), info.length());
+    return std::hash<std::string>{}(info);
```

Comment:
Use std::hash since the collision probability approaching 1.0/std::numeric_limits<size_t>::max().